### PR TITLE
Refactoring Formula tests

### PR DESCRIPTION
**Pull request by maxschlosser**
I found some parts of the tests for the ReferenceFormulas that I thought could be improved.

Please let me know what you think.

### CODE REVIEW

File: code/src/utest/pcgen/base/formula/AddingFormulaTest.java [hunk 1]
```diff
@@ -19,116 +19,107 @@
...
-		{
-			f.resolve(4, 2.5);
-			fail("two arguments should be illegal");
-		}
-		catch (IllegalArgumentException e)
-		{
-			// OK
-		}
-	}
+    private AddingFormula f;
```
Comment (karianna) [inline code/src/utest/pcgen/base/formula/AddingFormulaTest.java]: more descriptive variable name, please.

File: code/src/utest/pcgen/base/formula/AddingFormulaTest.java [hunk 2]
```diff
@@ -19,116 +19,107 @@
...
+    {
+        AddingFormula f = new AddingFormula(-2);
+        assertEquals(3, f.resolve(5).intValue());
+        assertEquals(-8, f.resolve(-6.7).intValue());
+    }
+
+    @Test
+    void testStackedFormula()
+    {
+        assertEquals(3, f.resolve(f.resolve(1)));
```
Comment (karianna) [inline code/src/utest/pcgen/base/formula/AddingFormulaTest.java]: whitespace inconsistency

File: code/src/utest/pcgen/base/formula/AddingFormulaTest.java (hunk 1 above)
Comment (karianna) [reply code/src/utest/pcgen/base/formula/AddingFormulaTest.java]: SAme goes for the other classes

**Comment by maxschlosser** [comment]
a6541f3 should include the requested changes

83dd50b seemed like a good idea to communicate that formula currently rounds like int multiplication

**Pull request merged**